pom.xml

```diff
@@ -23,7 +23,7 @@
 	<parent>
 		<groupId>de.fau.cs.osr</groupId>
 		<artifactId>tooling</artifactId>
-		<version>3.0.9-SNAPSHOT</version>
+		<version>3.0.8</version>
```

Comment:
That looks curious to me. I think it is normal practice to set the version to the next patch number followed by `-SNAPSHOT` to differentiate the development branch from an official release. Why did you need to do that?

Reply:
So in general I agree, that `-SNAPSHOT` needs to be added to indicate a development version (for the actual modules inside a project). However, I think, that a parent, which resides outside of the actual project, should be a released artifact to ease development and to avoid the obstacles mentioned above.